app/jobs/score_updater_job.rb

```diff
@@ -1,22 +1,22 @@
 class ScoreUpdaterJob
   include SuckerPunch::Job
 
-  def perform(user_id, score=1)
-    add_score(user_id, score, DailyScore)
-    add_score(user_id, score, WeeklyScore)
-    add_score(user_id, score, MonthlyScore)
-    add_score(user_id, score, OverallScore)
+  def perform(user_id, delta=1)
+    add_delta(user_id, delta, DailyScore)
+    add_delta(user_id, delta, WeeklyScore)
+    add_delta(user_id, delta, MonthlyScore)
+    add_delta(user_id, delta, OverallScore)
   end
 
-  def add_score(user_id, score, score_klass, expires_at=nil)
+  def add_delta(user_id, delta, score_klass, expires_at=nil)
```

Comment:
I kind of agree. I propose `inc_score!`. `inc` because it uses `Mongoid#inc` and `!` because it invokes `create!` to create a new record. The variable name could be changed as well, but I wouldn't like to call it `score`, any suggestion?